### ISSUE TITLE
Unable to find any GraphQL type definitions for the following pointers: ./app/**/*.{ts,tsx}

### ISSUE THREAD

**Issue by Albert-Gao** [graphql/vscode-graphql#228]
1. I use Relay, so there is no `.graphql` file in code, everything is in code
![image](https://user-images.githubusercontent.com/18282328/94207882-ad4ea100-ff1c-11ea-9a46-2fe949954730.png)
1. With a config like this,
```javascript
module.exports = {
  projects: {
    default: {
      schema: ['./schema.graphql'],
      documents: ['./app/**/*.{ts,tsx}'],
      endpoints: {
        default: {
          url: 'http://localhost:4000/graphql',
          headers: { Authorization: jwt },
        },
      },
    },
  },
}
```
3. It outputs error from GraphQL language server

![image](https://user-images.githubusercontent.com/18282328/94208010-f999e100-ff1c-11ea-8da9-5564e7b090ae.png)

## Specifications
GraphQL for VSCode Extension Version: latest from store
Version: 1.49.1
Commit: 58bb7b2331731bf72587010e943852e13e6fd3cf
Date: 2020-09-16T23:21:17.533Z
Electron: 9.2.1
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Darwin x64 19.6.0

**Comment by VictorGlindasPaf** [comment]
I'm in the same boat, so this seems to still be an issue.
My config is as such:
```js
// @ts-check
/**
 * @type {Partial<import("graphql-config").GraphQLProjectConfig>}
 */
const config = {
  name: 'Project',
  schema: './packages/schema-types/__generated__/schema.graphql',
  documents: ['./packages/**/*.{ts,tsx}'],
  extensions: {
    endpoints: {
      Next: {
        url: 'http://localhost:3000/api/graphql',
        headers: {
          'user-agent': 'JS GraphQL',
        },
        introspect: false,
      },
    },
  },
};

module.exports = config;
```

And the language service output:
```

4/1/2021, 11:14:09 AM [1] (pid: 5087) graphql-language-service-usage-logs: invalid/unknown file in graphql config documents entry:
 './packages/**/*.{ts,tsx}'
4/1/2021, 11:14:09 AM [1] (pid: 5087) graphql-language-service-usage-logs: Error: 
      Unable to find any GraphQL type definitions for the following pointers:
        
          - ./packages/**/*.{ts,tsx}
```

**Comment by tinleym** [comment]
Same issue here as well...

**graphql.config.js**
`module.exports = {
  schema: "./lib/data/schema.graphql",
  documents: "./lib/data/__generated__/*",
};`

**output**
`Error: graphql operation failed Error: Unable to find any GraphQL type definitions for the following pointers: - ./lib/data/__generated__/*`

Schema definitions appear to be working.

**Comment by tilman** [comment]
Had the same Issue and adding one dummy query in an .graphql file helped me resolve the Issue. So it seems that you need at least one .graphql file in your project as also described by this issue: https://github.com/graphql/vscode-graphql/issues/309

**Comment by sbking** [comment]
Same issue. Is this extension being maintained?

**Comment by acao** [comment]
@tilman we have no way of making template tag replacement for fragments work like that, sorry!

I would just suggest using the fragment name directly instead of using template tag replacement. The language server is set up to automatically complete for fragments throughout your codebase. Template tag replacement will break this

**Comment by acao** [comment]
@sbking yes i would say so since i just added this feature a month ago 😅

**Comment by acao** [comment]
We are waiting on @dotansimha and members of the guild to take over maintenance of the language sever and this reference implementation extension. Moving this to the appropriate repo
